scripts/expert/MinecraftGettingStarted.zs

```diff
@@ -2,220 +2,317 @@
-recipes.addShapeless(<forestry:planks.1:7> * 2, [<forestry:logs.5:3>]);
-recipes.addShapeless(<forestry:planks.1:6> * 2, [<forestry:logs.5:2>]);
-recipes.addShapeless(<forestry:planks.1:5> * 2, [<forestry:logs.5:1>]);
-recipes.addShapeless(<forestry:planks.1:4> * 2, [<forestry:logs.5>]);
-recipes.addShapeless(<forestry:planks.1:3> * 2, [<forestry:logs.4:3>]);
-recipes.addShapeless(<forestry:planks.1:2> * 2, [<forestry:logs.4:2>]);
-recipes.addShapeless(<forestry:planks.1:1> * 2, [<forestry:logs.4:1>]);
-recipes.addShapeless(<forestry:planks.1> * 2, [<forestry:logs.4>]);
-recipes.addShapeless(<forestry:planks.0:15> * 2, [<forestry:logs.3:3>]);
-recipes.addShapeless(<forestry:planks.0:14> * 2, [<forestry:logs.3:2>]);
-recipes.addShapeless(<forestry:planks.0:13> * 2, [<forestry:logs.3:1>]);
-recipes.addShapeless(<forestry:planks.0:12> * 2, [<forestry:logs.3>]);
-recipes.addShapeless(<forestry:planks.0:11> * 2, [<forestry:logs.2:3>]);
-recipes.addShapeless(<forestry:planks.0:10> * 2, [<forestry:logs.2:2>]);
-recipes.addShapeless(<forestry:planks.0:9> * 2, [<forestry:logs.2:1>]);
-recipes.addShapeless(<forestry:planks.0:8> * 2, [<forestry:logs.2>]);
-recipes.addShapeless(<forestry:planks.0:7> * 2, [<forestry:logs.1:3>]);
-recipes.addShapeless(<forestry:planks.0:6> * 2, [<forestry:logs.1:2>]);
-recipes.addShapeless(<forestry:planks.0:5> * 2, [<forestry:logs.1:1>]);
-recipes.addShapeless(<forestry:planks.0:4> * 2, [<forestry:logs.1>]);
-recipes.addShapeless(<forestry:planks.0:3> * 2, [<forestry:logs.0:3>]);
-recipes.addShapeless(<forestry:planks.0:2> * 2, [<forestry:logs.0:2>]);
-recipes.addShapeless(<forestry:planks.0:1> * 2, [<forestry:logs.0:1>]);
-recipes.addShapeless(<forestry:planks.0> * 2, [<forestry:logs.0>]);
-recipes.addShapeless(<extrautils2:ironwood_planks> * 2, [<extrautils2:ironwood_log>]);
-recipes.addShapeless(<extrautils2:ironwood_planks:1> * 2, [<extrautils2:ironwood_log:1>]);
-recipes.addShapeless(<thaumcraft:plank_greatwood> * 2, [<thaumcraft:log_greatwood>]);
-recipes.addShapeless(<thaumcraft:plank_silverwood> * 2, [<thaumcraft:log_silverwood>]);
-recipes.addShapeless(<natura:overworld_planks:5> * 2, [<extratrees:logs.8>]);
-recipes.addShapeless(<natura:overworld_planks:5> * 2, [<natura:overworld_logs2:1>]);
-recipes.addShapeless(<natura:overworld_planks:5> * 2, [<extratrees:logs.9>]);
-recipes.addShapeless(<extratrees:logs.fireproof.9> * 2, [<extratrees:logs.fireproof.9>]);
+#Recipe Remove for variant things
+    var WoodArray = [
+        # Log, Plank
+        [<minecraft:log>, <minecraft:planks>],
+        [<minecraft:log:1>, <minecraft:planks:1>],
+        [<minecraft:log:2>,<minecraft:planks:2>],
+        [<minecraft:log:3>,<minecraft:planks:3>],
+        [<minecraft:log2:1>,<minecraft:planks:5>],
+        [<minecraft:log2>,<minecraft:planks:4>],
+
+        [<extratrees:logs.0>, <extratrees:planks.0>],
+        [<extratrees:logs.0:1>, <extratrees:planks.0:1>],
```

Comment:
Sorry for being nit-picky, but isn't this too much indentation.

Yeah, just a nit pick, but indentation is all over the place.